FingerprintDialogKotlin/app/src/main/java/com/example/android/fingerprintdialog/MainActivity.kt

```diff
@@ -85,34 +89,25 @@ class MainActivity : AppCompatActivity(),
         val purchaseButtonNotInvalidated =
                 findViewById<Button>(R.id.purchase_button_not_invalidated)
 
-        purchaseButtonNotInvalidated.run {
-            isEnabled = true
-            setOnClickListener(PurchaseButtonClickListener(
-                    cipherNotInvalidated, KEY_NAME_NOT_INVALIDATED))
-        }
+        // migration before and after: keeping the same pattern of disabling buttons if
```

Comment:
nit: I'd rather have this comment describe what the code does _now_, rather than how it was migrated. Also, let's try to stick to sentence case for comments.